src/episode.py

```diff
@@ -0,0 +1,36 @@
+from telegram.ext import Updater
+import requests
+from bs4 import BeautifulSoup
+from telegram.ext import CommandHandler
+
+# Define podcast formats
```

Comment:
I would use a separate variable for the url and then just add the path afterwards. Probably best to do it in the config file.